docs/configuration.asciidoc

```diff
@@ -134,7 +134,7 @@ Below is a sample `Web.config` configuration file for a ASP.NET application.
 This is used to keep all the errors and transactions of your service together
 and is the primary filter in the Elastic APM user interface.
 
-NOTE: The service name must conform to this regular expression: ^[a-zA-Z0-9 _-]+$. In less regexy terms: Your service name must only contain characters from the ASCII alphabet, numbers, dashes, underscores and spaces. Characters in service name which don't match regular expression will be replaced by `__` symbol.
+NOTE: The service name must conform to this regular expression: `^[a-zA-Z0-9_-]+$`. In less regexy terms: Your service name must only contain characters from the ASCII alphabet, numbers, dashes, underscores and spaces. Characters in service name which don't match regular expression will be replaced by `__` symbol.
```

Comment:
I think we need to replace two underscores in "replaced by `__` symbol" above by just one underscore because while it does render as one underscore at github it renders as two underscores at https://www.elastic.co/guide/en/apm/agent/dotnet/current/config-core.html#config-service-name.

Reply:
Fixed in 1493271 (all numbers -- very cool)